src/components/elements/Map-mapbox/MapControls/CheckPolygonControl.tsx

```diff
@@ -40,6 +41,17 @@ const CheckPolygonControl = (props: CheckSitePolygonProps) => {
   const context = useSitePolygonData();
   const sitePolygonData = context?.sitePolygonData;
   const t = useT();
+  const [notificationStatus, setNotificationStatus] = useState<{
+    open: boolean;
+    message: string;
+    type: "success" | "error" | "warning";
+    title: string;
+  }>({
+    open: false,
+    message: "",
+    type: "success",
+    title: "Success!"
```

Comment:
Translation for titles here and on lines 91 and 96

Reply:
done